### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by twinie96**

### CODE REVIEW

File: app/models/place.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def full_address
 
   def start_date_within_place_dates
     if trip.start_date > start_date
```
Comment (Shaqal7) [inline app/models/place.rb]: ">="

File: app/views/trips/_current_trips.html.erb [hunk 2]
```diff
@@ -11,10 +11,10 @@
       <div class="col-lg-4">
         <h1><%= trip.name %></h1>
         <p>
-          From: <%= trip.start_date if trip.start_date %>
+          From: <%= trip.start_date.strftime("%d/%m/%Y") if trip.start_date %>
```
Comment (Shaqal7) [inline app/views/trips/_current_trips.html.erb]: we can delete if statement in view cause we required a start and end date

File: app/views/trips/_current_trips.html.erb [hunk 3]
```diff
@@ -11,10 +11,10 @@
...
         </p>
         <p>
-          Until: <%= trip.end_date if trip.end_date %>
+          Until: <%= trip.end_date.strftime("%d/%m/%Y") if trip.end_date %>
```
Comment (Shaqal7) [inline app/views/trips/_current_trips.html.erb]: same there

File: app/views/trips/_past_trips.html.erb [hunk 4]
```diff
@@ -12,10 +12,10 @@
           <div class="card-body">
             <h2 class="card-title"><%= trip.name %></h2>
             <p class="card-text">
-              From: <%= trip.start_date if trip.start_date %>
+              From: <%= trip.start_date.strftime("%d/%m/%Y") if trip.start_date %>
```
Comment (Shaqal7) [inline app/views/trips/_past_trips.html.erb]: and there

File: app/views/trips/_past_trips.html.erb [hunk 5]
```diff
@@ -12,10 +12,10 @@
...
             </p>
             <p class="card-text">
-              Until: <%= trip.end_date if trip.end_date %>
+              Until: <%= trip.end_date.strftime("%d/%m/%Y") if trip.end_date %>
```
Comment (Shaqal7) [inline app/views/trips/_past_trips.html.erb]: and there

File: app/views/trips/_upcoming_trips.html.erb [hunk 6]
```diff
@@ -12,10 +12,10 @@
           <div class="card-body">
             <h2 class="card-title"><%= trip.name %></h2>
             <p class="card-text">
-              From: <%= trip.start_date if trip.start_date %>
+              From: <%= trip.start_date.strftime("%d/%m/%Y") if trip.start_date %>
```
Comment (Shaqal7) [inline app/views/trips/_upcoming_trips.html.erb]: and there

File: app/models/place.rb (hunk 1 above)
Comment (twinie96) [reply app/models/place.rb]: Of course!

File: app/views/trips/_current_trips.html.erb (hunk 2 above)
Comment (twinie96) [reply app/views/trips/_current_trips.html.erb]: Totally right

**Pull request merged**